javascript/node/selenium-webdriver/phantomjs.js

```diff
@@ -15,6 +15,28 @@
 // specific language governing permissions and limitations
 // under the License.
 
+/**
+ * @fileoverview Defines a {@linkplain Driver WebDriver} client for the PhantomJS
+ * web browser. By default, it is expected that PhantomJS exists in your working directory.
```

Comment:
By default, it is expected that the PhantomJS executable can be located on your
    [PATH](https://en.wikipedia.org/wiki/PATH_(variable))